wfl/Database/MongoDBManager.cs

```diff
@@ -82,9 +57,9 @@ private string GenerateCollectionName<T>()
             return entity;
         }
 
-        public async Task<IEnumerable<T>> GetEntitiesByDateAsync<T>(DateTime date) where T : IEntity
+        public async Task<IEnumerable<T>> GetEntitiesByDateAsync(DateTime date)
```

Comment:
I think this is a reasonable approach for now and filtering by date since every entity is forced to have a create date via it's Id.  However, at some point we'll likely have to think about how to make filtering more generic and still flexible to support multiple objects.